### PR TITLE
Specify TS plugins options

### PR DESCRIPTION
**Pull request by rifler**

### CODE REVIEW

**Comment by rifler** [comment]
done. I am not very familiar with babel, can you review these changes to be accurate?

File: docs/preset-typescript.md [hunk 1]
```diff
@@ -58,11 +58,23 @@ require("@babel/core").transform("code", {
...
-`string`
+`string`, defaults to `React`.
 
 Replace the function used when compiling JSX expressions.
 
+### `allExtensions`
+
+`boolean`, defaults to `false`.
+
+If `true`, applies `jsxPragma` and `isTSX` options to both `.ts` and `.tsx` files. Otherwise `isTSX` option will be applied only to `.tsx` files.
```
Comment (nicolo-ribaudo) [inline docs/preset-typescript.md]: Actually this option indicates that every file (`.ts`, `.js`, `.custom-ext`, ...) should be parsed as TS (or TSX, depeding on the `isTSX` option).

File: docs/preset-typescript.md [hunk 2]
```diff
@@ -58,11 +58,23 @@ require("@babel/core").transform("code", {
...
 
 Replace the function used when compiling JSX expressions.
 
+### `allExtensions`
+
+`boolean`, defaults to `false`.
+
+If `true`, applies `jsxPragma` and `isTSX` options to both `.ts` and `.tsx` files. Otherwise `isTSX` option will be applied only to `.tsx` files.  
+
 This is so that we know that the import is not a type import, and should not be removed
```
Comment (nicolo-ribaudo) [inline docs/preset-typescript.md]: This line is for `jsxPragma`

**Comment by rifler** [comment]
fixed

**Pull request merged**